src/runtime_src/core/include/xgq_cmd_vmr.h

```diff
@@ -126,11 +126,17 @@ struct xgq_cmd_data_payload {
 	uint64_t address;
```

Comment:
Variable not initialized.

Reply:
this is the definition of struct xgq_cmd_data_payload in the header file.